src/index.js

```diff
@@ -90,16 +90,59 @@ function cropScreenshot([rect, screenshot]) {
  * @returns {Object|undefined}
  */
 function getElementBoundingRect(elementSelector) {
-    var element = document.querySelectorAll(elementSelector)[0];
-    if (element) {
+    function computeFrameOffset(win, dims) {
```

Comment:
It will be better to place ``computeFrameOffset`` and ``computeElementRect`` functions outside ``getElementBoundingRect`` and to add JSDoc to them